Extension/src/LanguageServer/Providers/documentFormattingEditProvider.ts

```diff
@@ -0,0 +1,66 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All Rights Reserved.
+ * See 'LICENSE' in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+import * as vscode from 'vscode';
+import { DefaultClient, FormatParams, FormatDocumentRequest, cachedEditorConfigSettings } from '../client';
+import { CppSettings } from '../settings';
+import * as editorConfig from 'editorconfig';
+
+
```

Comment:
There's a double newline here and in a few other files. If you feel like it, you could change the existing "no-multiple-empty-lines" in .eslintrc.js to be "no-multiple-empty-lines": ["error", { "max": 1, "maxEOF": 1, "maxBOF": 0}].
If not, we could do that in a separate PR.

Reply:
Okay sir